hcl2template/function/datetime.go

```diff
@@ -1,12 +1,22 @@
 package function
 
 import (
+	"fmt"
 	"time"
 
 	"github.com/zclconf/go-cty/cty"
 	"github.com/zclconf/go-cty/cty/function"
 )
 
+// InitTime is the UTC time when this package was initialized. It is
+// used as the timestamp for all configuration templates so that they
+// match for a single build.
+var InitTime time.Time
+
+func init() {
+	InitTime = time.Now().UTC()
+}
```

Comment:
Really cool !

Reply:
Yeah, this is lifted from the JSON function.